src/im_mad/im_exec/one_im_exec.rb

```diff
@@ -74,7 +74,7 @@ def action_monitor(number, host, ds_location, do_update)
         if !action_is_local?(:MONITOR)
             if do_update == "1" || @options[:force_copy]
                 # Use SCP to sync:
-                sync_cmd = "scp -r #{@local_scripts_base_path}/. " \
+                sync_cmd = "scp -r #{@local_scripts_base_path}/* " \
```

Comment:
If the destination directory `/var/tmp/one` is missing, the scp will fail (as the directory isn't explicitly created in this helper):

```
scp: /var/tmp/one: No such file or directory
```

There should be ssh+mkdir added.

```
sync_cmd = "ssh #{host['NAME']} mkdir -p '#{remote_dir}' 2>/dev/null &&" \
    "scp -rp #{REMOTES_LOCATION}/* " \
    "#{host['NAME']}:#{remote_dir} 2> /dev/null"
```